### ISSUE TITLE
Additional data source for Belgium / bring BE back

### ISSUE THREAD

**Issue by alixunderplatz** [tmrowco/electricitymap#916]
Hey guys!

Since the data on entso-e are incomplete / N/A for some of the generation types for **Belgium**, I suggest combining the data with generation schedules from the **Belgian TSO Elia**.
I have in mind to use the data for generation from **gas** and **hydro** from the schedule. Gas makes up about 1/3 of generation during the day, and hydro (probably pumped storage) up to 10%.

Schedules for the present day as well as for the previous ones are available as XLS and HTML downloads below the diagram on the site.

[http://www.elia.be/en/grid-data/power-generation/generation-schedules](http://www.elia.be/en/grid-data/power-generation/generation-schedules)

![image](https://user-images.githubusercontent.com/25743609/33993824-6da949f2-e0d8-11e7-99b1-9e40e94b0586.png)

There are also "actual generation"-data for the previous day, which seem to be valid for the past.
[http://www.elia.be/en/grid-data/power-generation/energy-cipu-units](http://www.elia.be/en/grid-data/power-generation/energy-cipu-units)

I compared data for yesterday's scheduled and actual data for gas and hydro:
![image](https://user-images.githubusercontent.com/25743609/33995398-2a82ac94-e0de-11e7-8350-97611f80ae6e.png)
and
![image](https://user-images.githubusercontent.com/25743609/33995435-47ee2d8a-e0de-11e7-865c-a21cd1162122.png)

Total generation also matches the load of about 11 to 12 GW during daytime quite well. I think it is precise enough to put the scheduled data on the map. What do you think?

**Comment by corradio** [comment]
Sounds reasonable. @Dalius-ENTSO-E do you have any insights as to why BE data is missing on ENTSOE before we move on to use another datasource?

**Comment by systemcatch** [comment]
@alixunderplatz So far we've avoided using scheduled or predicted generation.  But the past day breakdown would be useful to have for the dataset while the ENTSOE feed is not working.

**Comment by Dalius-ENTSO-E** [comment]
gas, hard coal, oil, all hydro, waste are missing since 12.12, biomass since 21.11. Request to Elia sent, hope that issue will be solved soon.

**Comment by alixunderplatz** [comment]
@systemcatch right, I was also thinking of rather using this for the dataset in case of unavailable entso-e data, since they might still be imported from Elia then.

**Comment by lmbxl** [comment]
Hello,

I noticed that no BE data appear anymore in the Electricity Map 
(https://www.electricitymap.org/?wind=false&solar=false&page=country&countryCode=BE) 
although several BE data (nuclear, other, solar, wind offshore, wind onshore *) are published on the ENTSO-E transparency platform (https://transparency.entsoe.eu/generation/r2/actualGenerationPerProductionType/show).

If the ENTSO-E transparency platform populates the electricity map (via Rest API) once-a-day, would anybody know the reason why the nuclear, other, solar, wind offshore and wind onshore BE data are not published in the Electricity Map anymore?
----------
(*) Data are available after 3 days on the ENTSO-E transparency platform and only data for the last 2 days are missing. The cause has been found and the resolution is ongoing. I will keep you posted as soon as all data will be available again.

**Comment by corradio** [comment]
Hi @lmbxl,
The Electricity Map shows data for the last 2 **hours**, which is why it doesn't appear. We actually update data from ENTSOE every 5 minutes.
It worked however in the past -- do you know if anything changed?

**Comment by lmbxl** [comment]
Hello Corradio,

Problem seems resolved at present :-) 
BE data are available again. 

The data of the generations by type were no more published in real time (J; -J + 2) as the associated mnemonics had been modified in the system source.
